Service/Form/Submission/FormSubmissionRequest.php

```diff
@@ -80,9 +80,9 @@ public function getLabel(): string
         return $this->label;
     }
 
-    public function getDeadlineDate(): string
+    public function getDeadlineDate(): ?\DateTime
     {
-        return $this->deadlineDate;
+        return $this->deadlineDate !== false ? $this->deadlineDate : null;
```

Comment:
It is never assigne to false (as far as I have reviewed your code)

Reply:
Same here, Travis gives an error like "deadline date cannot be DateTime|false|null".